src/main/webapp/app/exercises/shared/feedback/feedback-service.ts

```diff
@@ -28,6 +28,11 @@ export class FeedbackService {
      * @param result
      */
     public getFeedbacksForResult(participationId: number, result: Result): Observable<Feedback[]> {
-        return this.resultService.getFeedbackDetailsForResult(participationId, result).pipe(map(({ body: feedbackList }) => feedbackList!));
+        return this.resultService.getFeedbackDetailsForResult(participationId, result).pipe(
```

Comment:
I think this is not necessary since it is already done in the resultService

```ts
    getFeedbackDetailsForResult(participationId: number, result: Result): Observable<HttpResponse<Feedback[]>> {
        return this.http.get<Feedback[]>(`${this.participationResourceUrl}/${participationId}/results/${result.id!}/details`, { observe: 'response' }).pipe(
            map((res) => {
                const feedbacks = res.body ?? [];
                feedbacks.forEach((feedback) => (feedback.result = result));
                return res;
            }),
        );
    }```

Reply:
:+1: removed